app/src/main/java/com/github/se/polyfit/viewmodel/map/MapViewModel.kt

```diff
@@ -0,0 +1,47 @@
+package com.github.se.polyfit.viewmodel.map
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.github.se.polyfit.data.remote.firebase.PostFirebaseRepository
+import com.github.se.polyfit.model.post.Location
+import com.github.se.polyfit.model.post.Post
+
+class MapViewModel(private val repository: PostFirebaseRepository = PostFirebaseRepository()) :
```

Comment:
In order to be inline with the rest of the code base, it might be a good idea to use a hilt view model. This can be done by simply annotating the function with the tag `@HiltViewModel`.

Reply:
Thank you.
added